### DISCUSSION TITLE
Hide taiko-slider when using cinema mod

### DISCUSSION THREAD

**Discussion by Joppe27** [ppy/osu#15866] [Ideas] [answered]
The goal of the cinema mod is being able to watch the video that comes with a beatmap. The taiko-slider and taiko-slider-fail elements should probably be hidden (stable behavior) as they cover up the video which defeats the point of the cinema mod: 
![osu_2021-11-29_22-07-56](https://user-images.githubusercontent.com/56885706/143953354-55b904f0-e489-45d9-ba06-8e0f276e873a.jpg)
This was probably overlooked/forgotten because the osu!lazer default skin doesn't use this element anymore.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/15869
